### ISSUE TITLE
CogVideoX-2B SFT diffuser

### ISSUE THREAD

**Issue by HaoZhang990127** [THUDM/CogVideo#309]
### System Info / 系統信息

...

### Information / 问题信息

- [ ] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

how can I sft a transformer using diffuer?

### Expected behavior / 期待表现

Hi,

Thank you for your nice work, for the diffuer finetunning code, i see the lora version, you see you have _CogVideoX-2B SFT: 8 * A100_ version, could you tell me how can I sft a transformer using diffuer?

<img width="1049" alt="image" src="https://github.com/user-attachments/assets/826a4f3a-5f36-49b6-b604-2b2f2607ddaa">

Thank you

**Comment by zRzRzRzRzRzRzR** [comment]
For SFT we are still working, soon with doc and code here, now net some problem

**Comment by SHYuanBest** [comment]
Are the scheduler, loss, and training methods used in the diffusers version of finetuning the same as those used in training the base model? Is it just training LoRa instead?

**Comment by zZer0o** [comment]
> Are the scheduler, loss, and training methods used in the diffusers version of finetuning the same as those used in training the base model? Is it just training LoRA instead?

same questions here

**Comment by zRzRzRzRzRzRzR** [comment]
The learning rate should be reduced to 1E-5, the other parts remain the same, and in SAT's SFT and Lora, the difference is only in this aspect
